src/index.ts

```diff
@@ -14,14 +14,29 @@ class PgTransport extends Transform {
     this.table = table
     this.column = column
     this.client = client
+
+    process.stdin.on('end', () => this._shutdown('input ended'))
+    process.on('SIGINT', () => this._shutdown('SIGINT'))
+    process.on('SIGTERM', () => this._shutdown('SIGTERM'))
+  }
+
+  _shutdown (reason: string) {
```

Comment:
No need for a reason in there until we need it somewhere.

Reply:
this is from some code I copied from my pino logger.  Removing....